tonic/handler.go

```diff
@@ -195,10 +195,10 @@ func bind(c *gin.Context, v reflect.Value, tag string, extract extractor) error
 		}
 		// Set-up context for extractors.
 		// Query.
-		c.Set(ExplodeTag, true) // default
-		if explodeVal, ok := ft.Tag.Lookup(ExplodeTag); ok {
-			if explode, err := strconv.ParseBool(explodeVal); err == nil && !explode {
-				c.Set(ExplodeTag, false)
+		c.Set(ArrayTag, false)
```

Comment:
The OpenAPI 3 spec defines the default serialization to be true by default for `explode`.
> The default serialization method is style: form and explode: true.

Should we follow that ?

Reply:
That's actually the case here. The whole explode true/false semantics are insanely confusing.
But having the comma-separated list split logic off by default ends up meaning the same thing as explode=true by default.